app/components/Board.js

```diff
@@ -57,11 +57,21 @@ export default class Board extends Component {
   }
 
   render() {
+    var userId = "philzhan";
+
+    if ( require('getmac').isMac("ac:de:48:00:11:22") ) {
```

Comment:
isMac() doesn't work becuase it returns true for any valid mac address, not just for your your mac address. 
![image](https://user-images.githubusercontent.com/20145487/54650699-1d59ac80-4a86-11e9-8a5b-0e08195edfd8.png)
![image](https://user-images.githubusercontent.com/20145487/54650719-2d718c00-4a86-11e9-9a97-a181a54c596e.png)